docs/package.json

```diff
@@ -16,9 +16,11 @@
     "lint:js": "eslint --ignore-path .gitignore --fix .",
     "lint:css": "stylelint \"src/**/*.css\"",
     "lint": "npm-run-all --parallel lint:*",
+    "rebuild-phenomic": "cd ../ && npm -s run transpile",
```

Comment:
This was causing me issue when working on cache/dll, maybe I should do that now :)